applications/vanilla/models/class.categorymodel.php

```diff
@@ -2534,10 +2534,13 @@ public static function permissionCategory($category) {
         if (!is_array($category) && !is_object($category)) {
             $category = self::categories($category);
         }
-
         $category = self::categories(val('PermissionCategoryID', $category));
+
         // Ensure all of our values are processed properly.
-        self::calculate($category);
+        if (isset($category)) {
```

Comment:
A few lines up, we're returning the default category for permissions if the target category,`$category`, doesn't exist (is "empty"). What do you think about returning the default category for permissions if the actual "permission category" doesn't exist?